docs/MSPX.md

```diff
@@ -24,28 +26,87 @@ MspX is enabled by this setting in the receiver:
 
 For enabling MSP-RC set:
 
-- "Rx Snd RcChannel" = "rc override" or "rc channels"
+- "Rx Snd RcChannel" = "rc override" or "rc channels" (both have the same functionality in mspX mode)
 
 With MSP-RC enabled, SET_RAW_RC messages are sent to the flight controller. In the flight controller configure the receiver to the MSP protocol. This allows one to avoid the extra wire for CRSF or SBus. MspX needs to be enabled.
 
+***Notes***:
+RC Link Statistics are only send to the Flight controller via MSP, if "rc override" or "rc channels" is enabled. 
+MSP-RC messages will always override other RC Link inputs. If CRSF is connected and CRSF selected as the RC Protocol in INAV, MSP-RC still has priority if enabled in mLRS and connected to a MSP Enabled UART.
+
 ## INAV Configuration
 
-The baudrate should be set to 115200 as lower baudrates have shown irregular telemetry update rates on the radio due to the lack of flow control on the INAV side. 
+The baudrate should be set to 115200 or higher for consistent dataflow and RC control. Baudrates of less than 57600 can cause message loss on the INAV side and result in a inconsistent RC control at 50Hz RC Rate, if MSP-RC is used (INAV Limitation). 
```

Comment:
I think the Receiver configuration should also have this baud rate recommended.  It seems weird IMO that we're supposedly moving onto INAV but after reading this would first have to go back and change the baud rate on the receiver.

Reply:
Added that, good catch.